.github/release-drafter.yml

```diff
@@ -0,0 +1,6 @@
+template: |
```

Comment:
Do we want to make use of the `change-template` option so merged PRs in the changes section look the same as they do in the PBS release notes? It appears the author is listed after the title and PR number by default and we omitted that for PBS:
`change-template: '* $TITLE (#$NUMBER)'`

Reply:
I don't really have a strong opinion about that one but I am happy to add it if you think that looks better